<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DrawToolsWidget</class>
 <widget class="QWidget" name="DrawToolsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>99</width>
    <height>143</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QLabel" name="title_lbl">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
font: bold 10px;
}</string>
     </property>
     <property name="text">
      <string>DRAW TOOLS</string>
     </property>
     <property name="margin">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="schrodinger::sketcher::SetAtomWidget" name="set_atom_wdg" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="charge_layout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="explicit_h_btn">
       <property name="toolTip">
        <string>Add or remove explicit hydrogens</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton { border : none; }</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../sketcher.qrc">
         <normaloff>:/icons/explicit_h.png</normaloff>:/icons/explicit_h.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="increase_charge_btn">
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Increase Charge</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton { border : none; }</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../sketcher.qrc">
         <normaloff>:/icons/increase_charge.png</normaloff>:/icons/increase_charge.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">charge_group</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="decrease_charge_btn">
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Decrease Charge</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton { border : none; }</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../sketcher.qrc">
         <normaloff>:/icons/decrease_charge.png</normaloff>:/icons/decrease_charge.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">charge_group</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="bond_frame">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="1" column="1">
       <widget class="schrodinger::sketcher::ModularToolButton" name="bond_query_btn">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton { border : none; }</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../sketcher.qrc">
          <normaloff>:/icons/aromatic_bond.png</normaloff>:/icons/aromatic_bond.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bond_group</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="schrodinger::sketcher::ModularToolButton" name="stereo_bond1_btn">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton { border : none; }</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../sketcher.qrc">
          <normaloff>:/icons/wedge_bond.png</normaloff>:/icons/wedge_bond.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bond_group</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QToolButton" name="single_bond_btn">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Single Bond</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton { border : none; }</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../sketcher.qrc">
          <normaloff>:/icons/single_bond.png</normaloff>:/icons/single_bond.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bond_group</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="schrodinger::sketcher::ModularToolButton" name="stereo_bond2_btn">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton { border : none; }</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../sketcher.qrc">
          <normaloff>:/icons/dashed_bond.png</normaloff>:/icons/dashed_bond.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bond_group</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="atom_chain_btn">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Bond Chain - Drag on canvas to draw a chain of carbons</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton { border : none; }</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../sketcher.qrc">
          <normaloff>:/icons/atom_chain.png</normaloff>:/icons/atom_chain.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bond_group</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="schrodinger::sketcher::ModularToolButton" name="bond_order_btn">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton { border : none; }</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../sketcher.qrc">
          <normaloff>:/icons/double_bond.png</normaloff>:/icons/double_bond.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bond_group</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>schrodinger::sketcher::ModularToolButton</class>
   <extends>QToolButton</extends>
   <header>schrodinger/sketcher/modular_tool_button.h</header>
  </customwidget>
  <customwidget>
   <class>schrodinger::sketcher::SetAtomWidget</class>
   <extends>QWidget</extends>
   <header>schrodinger/sketcher/set_atom_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../sketcher.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="bond_group"/>
  <buttongroup name="charge_group"/>
 </buttongroups>
</ui>
